src/main/webapp/PlaceGuideOnList.js

```diff
@@ -0,0 +1,41 @@
+class PlaceGuideOnList {
+  constructor(placeGuideId, place, name, creator, description, audioKey, audioLength, imageKey, createdByCurrentUser) {
+    const placeGuideDiv = document.createElement("div");
+    placeGuideDiv.setAttribute('id', placeGuideId);
+    
+    placeGuideDiv.addEventListener('click', function() {
+      PlaceGuideManager.highlightPlaceGuide(placeGuideId);
```

Comment:
highlightPlaceGuide is not a static function. 
I will create one placeGuideManager global variable for each page, please use that one.

Reply:
changed to placeGuideManager.